pkg/windows/buildenv/salt-cp.bat

```diff
@@ -2,10 +2,18 @@
 :: Script for starting the Salt-Minion
 :: Accepts all parameters that Salt-Minion Accepts
 
+:: Get current codepage, change codpage to Unicode
+@for /f "tokens=2 delims=:." %%x in ('chcp') do set cp=%%x
+@chcp 65001>nul
```

Comment:
The @ symbol for your additions is unnecessary because all the scripts start with `@echo off`. They can be safely removed without changing the behavior of the script.